src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,55 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Filters } from '../../types/Filtres';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string;
+  setQuery: Dispatch<SetStateAction<string>>;
+  status: Filters;
+  setStatus: Dispatch<SetStateAction<Filters>>;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = props => {
+  const { query, setQuery, setStatus } = props;
```

Comment:
The `status` prop is declared in the `Props` type but is not destructured or used in the component. Consider using it if needed or remove it from the `Props` type if it's unnecessary.